cmd/cueckoo/cmd/importpr.go

```diff
@@ -140,13 +148,25 @@ func importPRDef(c *Command, args []string) error {
 	); err != nil {
 		return err
 	}
-	if _, err := run(ctx, "git", "branch", "--set-upstream-to", originBaseRef); err != nil {
-		return err
-	}
 	log.Printf("rebased and squashed on %s", rebaseMsg)
 
 	// TODO: fix up common commit message issues, especially when squashing, in Go code.
-	// TODO: automate adding "Closes #PR as merged.".
+
+	// Add "Closes #PR as merged." Not that running this command will also end
+	// up adding a Change-Id trailer if the user has git commit hooks set for
+	// pre-commit. This is a good thing.
```

Comment:
note that adding the change-id was already happening via the "git commit --amend" below - I imagine you're aware, and that this is for the sake of better docs. just want to flag that this particular behavior should remain unchanged.

Reply:
True, but the noticeable difference here is that the change-id is now visible during the phase of editing the commit message whereas before it was not. Reason being (at least on my setup) is that the change-id is added by a post-edit commit hook. Hence that commit hook now fires as a result of the auto-edit, and the change-id appears in the editor during the final-phase edit.

Reply:
I've fixed the comment to be correct. I mentioned pre-commit, but it's actually post-commit.